sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/ReadEventOptions.cs

```diff
@@ -29,6 +29,8 @@ public class ReadEventOptions
         ///
         /// <value>The relative priority to associate with an exclusive reader; for a non-exclusive reader, this value should be <c>null</c>.</value>
         ///
+        /// <seealso cref="EventHubsException.FailureReason.ConsumerDisconnected"/>
```

Comment:
We may also want to consider a `<seealso` for the exception type itself above this.